package.json

```diff
@@ -47,7 +47,7 @@
     "mkamioner <mkamioner@gmail.com>"
   ],
   "scripts": {
-    "system-test": "repo-tools test run --cmd mocha -- system-test/*.js --no-timeouts",
+    "system-test": "mocha system-test/*.js --no-timeouts --bail",
```

Comment:
I think `repo-tools test run ...` is the right command. That's the one that nodejs-storage uses: https://github.com/googleapis/nodejs-storage/blob/4da861fc7c03fa4b4539966539937f96e66bf757/package.json#L56

Reply:
Agree, not sure why it got edited this way. Reverted.